src/test/kotlin/it/pagopa/aca/utils/AcaUtilsTests.kt

```diff
@@ -0,0 +1,97 @@
+package it.pagopa.aca.utils
+
+import it.pagopa.aca.ObjectTestUtils
+import it.pagopa.aca.domain.Iupd
+import it.pagopa.generated.gpd.model.PaymentPositionModelBaseResponseDto
+import java.util.stream.Stream
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import org.mockito.junit.jupiter.MockitoExtension
+
+@ExtendWith(MockitoExtension::class)
+@OptIn(ExperimentalCoroutinesApi::class)
+class AcaUtilsTests {
+
+    companion object {
+        private val acaUtils = AcaUtils()
+        const val creditorInstitutionCode = "77777777777"
+        const val iuv = "302001069073736640"
+        val iupd = Iupd(creditorInstitutionCode, iuv)
+
+        @JvmStatic
+        private fun validStatusForExecuteOperation() =
+            Stream.of(
+                PaymentPositionModelBaseResponseDto.StatusEnum.PUBLISHED,
```

Comment:
I would extends this stream to all statuses and add a second argument with the expected outcome so that the two below methods can be collapsed to only one

Reply:
fixed with 👉🏻 d74b4fe1b2c86e72004fd21520ce3c2fb1e35005